test/controllers/sessions_controller_test.rb

```diff
@@ -1,5 +1,61 @@
 require "test_helper"
 
 describe SessionsController do
+	describe "create" do
+		it "succeeds and redirects user upon logging in" do
+			# Arrange
```

Comment:
Why is this using tabs for indentation? Please change your editor settings to use two spaces, to make group projects go smoother.